INSTALL.md

```diff
@@ -3,25 +3,21 @@
 ## The index meta-database
 
 This package may be used to setup and run an [OPTIMADE index meta-database](https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst#index-meta-database).
-Install the package via `pip install optimade[server]` and change the file [`server.cfg`](server.cfg) found in the root of the package.
+Install the package via `pip install optimade[server]`.
+
+The preferred method of configuring the server is to use a [JSON file](example_json.config) and setting the `optimade_config_file` environment variable to point to the location of this JSON file. The server can also be configured via environmet variables prefixed with `optimade_` and the corresponding variable names in `ServerConfig` in `optimade/server/config.py`. These environment variables are not case sensitive, so both `OPTIMADE_CONFIG_FILE` and `optimade_config_file` are valid.
```

Comment:
```suggestion
The preferred method of configuring the server is to use a [JSON file](example_json.config) and setting the `optimade_config_file` environment variable to point to the absolute location of this JSON file.
The server can also be configured via environment variables prefixed with `optimade_` and the corresponding variable names in `ServerConfig` in the [`config.py` file](optimade/server/config.py).
These environment variables are not case sensitive, so both `OPTIMADE_CONFIG_FILE` and `optimade_config_file` are valid.
```

Reply:
There needs to be a mention of the `index_links_path` or `optimade_index_links_path` variable here as well - since this is crucial information for setting up the index meta-database.